src/frontend/src/test-e2e/views.ts

```diff
@@ -153,7 +153,7 @@ export class MainView extends View {
   async waitForDisplay(): Promise<void> {
     await this.browser
       .$("//h1[string()='Anchor Management']")
-      .waitForDisplayed({ timeout: 5_000 });
+      .waitForDisplayed({ timeout: 10_000 });
```

Comment:
did things get much slower?

Reply:
A bit, yes. This is because we switched from a query call to an update call.